.gitignore

```diff
@@ -0,0 +1,4 @@
+lib/*.bundle
+ext/debug_inspector/Makefile
+ext/debug_inspector/*.bundle
+ext/debug_inspector/*.o
```

Comment:
I think below is better.
"/" means top directory of this project. and "without /" means adapted for any directory.

```
*.o
/.bundle
/Gemfile.lock
Makefile
```